client/src/entrypoints/admin/preview-panel.js

```diff
@@ -18,7 +18,7 @@ const runContentChecks = async () => {
   axe.registerPlugin(wagtailPreviewPlugin);
 
   const contentMetrics = await getPreviewContentMetrics({
-    targetElement: 'main, [role="main"], body',
+    targetElement: 'main, [role="main"]',
```

Comment:
Can't test this part, which is actually where the bug is, due to it being in an unexported function. We could likely test this once #10356 is merged.